docs/buildah-build.1.md

```diff
@@ -96,7 +96,37 @@ The value of `[name]` is matched with the following priority order:
 
 **--cache-from**
 
-Images to utilise as potential cache sources. Buildah does not currently support --cache-from so this is a NOOP.
+Repository to utilise as potential cache source. When specified buildah will try to look for
+cache images in specified repository and will attempt to pull cache images instead of actually
+executing and computing STEPS. Buildah will only attempt to pull cache images if they are considered
+as valid cache hits otherwise not.
```

Comment:
I think, but don't know, that it probably ought to be "cached images" instead of "cache images" above.  @nalind ?

Reply:
Uh, "previously cached", I guess.